package.json

```diff
@@ -4,28 +4,31 @@
   "description": "Vanilla Framework, built using React",
   "main": "build/index.js",
   "scripts": {
+    "build": "npm-run-all build-css build-js",
     "build-css": "node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/",
-    "watch-css": "npm run build-css && node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/ --watch --recursive",
-    "start-js": "react-scripts start",
-    "start": "npm-run-all -p watch-css start-js",
     "build-js": "react-scripts build",
-    "build": "npm-run-all build-css build-js",
-    "test": "node scripts/test.js --env=jsdom",
-    "preserve": "yarn install && yarn run build",
-    "serve": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "snapshot": "build-storybook -s public && percy-storybook --widths=320,1280",
     "clean": "rm -rf node_modules && rm -rf build",
-    "lint": "eslint ./src/"
+    "lint": "eslint ./src/",
+    "watch-css": "npm run build-css && node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/ --watch --recursive",
+    "preserve": "yarn install && yarn run build",
+    "serve": "npm-run-all -p watch-css serve-storybook",
+    "serve-storybook": "start-storybook -p ${PORT:-8102}",
+    "snapshot": "build-storybook -s public && percy-storybook --widths=320,1280",
+    "test": "node scripts/test.js --env=jsdom"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/vanilla-framework/vanilla-framework-react.git"
   },
+<<<<<<< HEAD
+=======
+  "author": "Canonical webteam <webteam@canonical.com>",
+>>>>>>> Update package.json author and URLs
```

Comment:
@WillMoggridge I think maaaaaaaaybe this is why the CI is currently failing

Reply:
Woah, what is with everyone putting me on blast?!